### DISCUSSION TITLE
Sonoff Basic rev2 + BME280 - standalone

### DISCUSSION THREAD

**Discussion by padziadziu** [arendst/Tasmota#11000] [Q&A] [answered]
Hi All,

have a question
I've setup the Sonoff Basic rev 2 with i2c BME280 - workin just fine, sending all data via MQTT to my HA.
But I want it working standalone - mean - turn on the internal relay when the humidity is higher then some poin, and turn of then it is below another point - not using any external client/software etc.
Is there an option to setup tasmota to do this such easy automation?

Mat

**Comment by Jason2866** [comment, marked as answer]
Yes, via Rules. See examples for similar use case [here](https://tasmota.github.io/docs/Rules/#rule-cookbook)

**Comment by padziadziu** [comment]
Thanks for the hint

I'm newbee here...
any link to example how to "rule" the state of switch depends on humidity point (55%)?
I can do it using Domoticz events, (I'd like to have this working no matter Domoticz is running) but no idea how to set it up using rules. one example will help me a lot, please :)

like:
if "state of sensorYY"=>XX trigger switch_XX"on"
else trigger switch_XX"off"

Mat

**Reply by ascillato** [reply]
Those examples are already written in the Tasmota docs. Use the link that Jason have provided to you above. Thanks.
